roles/network/templates/vxlan.network.j2

```diff
@@ -3,6 +3,13 @@ Name={{ item.1.name }}
 
 [Network]
 IPv6AcceptRA=no
+{% if 'addresses' in item.1 -%}
+{%  for address in item.1.addresses %}
+
+[Address]
```

Comment:
using a dedicated section is only needed when adding parameters to some of them. in this case you can simply add all of them in the `Network` section

Reply:
Done, I actually missed that part in the docs